build/yamls/antrea-windows-with-ovs.yml

```diff
@@ -4,39 +4,12 @@ data:
     $ErrorActionPreference = "Stop"
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
-    $OVSDriverDir = "$mountPath\openvswitch\driver"
-
-    # Check if OVSExt driver is already installed
-    $driverStatus = netcfg -q ovsext
-    if ($driverStatus -like '*not installed*') {
-      # Install OVS Driver
-      $result = netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
-      if ($result -like '*failed*') {
-        Write-Host "Failed to install OVSExt driver: $result"
-        exit 1
-      }
-      Write-Host "OVSExt driver has been installed"
-    }
-
-    # Check if the VC redistributable is already installed.
-    $OVSRedistDir="$mountPath\openvswitch\redist"
-    if (Test-Path $OVSRedistDir) {
-      $dllFound = $false
-      $paths = $env:PATH -split ';'
-      foreach ($path in $paths) {
-        $dllFiles = Get-ChildItem -Path $path -Filter "vcruntime*.dll" -File -ErrorAction SilentlyContinue
-        if ($dllFiles.Count -gt 0) {
-          $dllFound = $true
-          break
-        }
-      }
-
-      # vcruntime dlls are not installed on the host, then install the binaries.
-      if (-not $dllFound) {
-        Get-ChildItem $OVSRedistDir -Filter *.exe | ForEach-Object {
-          Start-Process -FilePath $_.FullName -Args '/install /passive /norestart' -Verb RunAs -Wait
-        }
-      }
+    $OVSInstallScript = "$mountPath\k\antrea\Install-OVS.ps1"
+    if (Test-Path $OVSInstallScript) {
+      & $OVSInstallScript -InstallUserspace $false -LocalFile $mountPath\openvswitch
+    } else {
+      Write-Host "Installation script not found: $OVSInstallScript, please manually install OVS drivers"
```

Comment:
This would indicate that there is a version mismatch between the YAML manifest and the container image? I am not sure the error message is appropriate for this case.
Also, it seems that even if the user has installed the OVS driver manually, we will still fail with this error. This is because we just check for the script existence and not for the installation status of the driver.

Reply:
> This would indicate that there is a version mismatch between the YAML manifest and the container image?

Reply:
I fully agree with not duplicating code, but we should not imply in the error message that the issue can be resolved by manually installing the OVS driver. Would the following message work:

>       Write-Host "Installation script not found: $OVSInstallScript, you may be using an invalid antrea-windows container image"